.gitignore

```diff
@@ -103,3 +100,12 @@ WordPress/Credentials/Secrets.swift
 WordPress/Credentials/wpcom_app_credentials
 WordPress/Credentials/wpcom_alpha_app_credentials
 WordPress/Credentials/wpcom_internal_app_credentials
+# These headers, too, are not longer used or generated by the build process,
+# but we wnt to keep ignoring them to avoid them being accidentally committed.
+#
+# We shall remove this once enough time has passed since they became unused.
+# Maybe a couple of years would be enough for most developers to change machine
+# and no longer have these files in their local checkout.
+WordPress/InfoPlist.h
+WordPress/InfoPlist-alpha.h
+WordPress/InfoPlist-internal.h
```

Comment:
If you check your local copy, you'll likely have these files and they'll be empty.